guides/common/modules/con_provisioning-workflow.adoc

```diff
@@ -1,26 +1,5 @@
-[[Introduction]]
-== Introduction
-
-Provisioning is a process that starts with a bare physical or virtual machine and ends with a fully configured, ready-to-use operating system. Using {ProjectName}, you can define and automate fine-grained provisioning for a large number of hosts.
-
-There are many provisioning methods. For example, you can use {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer} to provision bare metal hosts using both PXE based and non-PXE based methods. You can also provision cloud instances from specific providers through their APIs. These provisioning methods are part of the {ProjectNameX} application life cycle to create, manage, and update hosts.
-
-[[Introduction-Defining_Provisioning_Types]]
-=== Provisioning Types Overview
-
-{ProjectNameX} has different methods for provisioning hosts:
-
-Bare Metal Provisioning::
-  {Project} provisions bare metal hosts primarily through PXE boot and MAC address identification. You can create host entries and specify the MAC address of the physical host to provision. You can also boot blank hosts to use {Project}'s discovery service, which creates a pool of ready-to-provision hosts. You can also boot and provision hosts through PXE-less methods.
-
-Cloud Providers::
-  {Project} connects to private and public cloud providers to provision instances of hosts from images that are stored with the Cloud environment. This also includes selecting which hardware profile or flavor to use.
-
-Virtualization Infrastructure::
-  {Project} connects to virtualization infrastructure services such as {OVirt} and VMware to provision virtual machines from virtual image templates or using the same PXE-based boot methods as bare metal providers.
-
-[[Introduction-Defining_the_Provisioning_Workflow]]
-=== Provisioning Workflow Overview
+[id="provisioning-workflow_{context}"]
+= Provisioning Workflow
 
 The provisioning process follows a basic PXE workflow:
 
```

Comment:
On the next line, change Satellite to {Project}:

. You create a host and select a domain and subnet. {Project} requests an available IP address from the DHCP {SmartProxyServer} that is associated with the subnet or from the PostgreSQL database in Satellite.

GitHub won't let me comment on this :-(